docs/tutorials/agility/agility.collection.json

```diff
@@ -12,7 +12,10 @@
 					"listen": "test",
 					"script": {
 						"exec": [
-							"pm.environment.set(\"access_token\", pm.response.json().access_token);"
+							"pm.environment.set(\"access_token\", pm.response.json().access_token);\r",
+							"const date = new Date();\r",
+							"const isoDate = date.toISOString();\r",
+							"pm.environment.set('currentDate', isoDate.substring(0, isoDate.length - 5) + 'Z');"
```

Comment:
Decimal seconds are supported in the required date format, so you can use the [Postman built-in dynamic variable](https://learning.postman.com/docs/writing-scripts/script-references/variables-list/) `$isoTimestamp` directly in the body of the two "Issue..." requests in this Agility collection, e.g., instead of:

```javascript
  "options": {
    "type": "Ed25519Signature2018",
    "created": "{{currentDate}}"
  }
```
you can use:

```javascript
  "options": {
    "type": "Ed25519Signature2018",
    "created": "{{$isoTimestamp}}"
  }
```